db/schema.rb

```diff
@@ -25,6 +47,7 @@
     t.datetime "remember_created_at"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.integer "role", default: 0
```

Comment:
Fajny przykła tego co sie dzieje, jak skaczesz po branchach gdzie tworzyles migracje,
Baze danych mamy jedną, będąc na branchu user_roles stworzyles migracje i baza sie zupdatowała o dodatkowe pole
Tworząc nowego brancha z mastera baza danych dalej posiada te migracje stworzoną wczesniej, więc na tym branchu jak zrobiłeś
rails db:migrate wrzuciło Ci zmiany do schemy, których nie powinno tutaj być, możesz to ręcznie usunąć teraz

A ogolnie przechodząc pomiędzy branchami gdzie robilismy migracje, dobrze jest sobie robic rails db:rollback

np. będąc na user_roles stworzyles migracje, chcesz stworzyc nowego brancha na masterze
robisz rails db:rollback
usuwasz zmiany na schemie, bo nie chcesz wypychać na githuba zmian po rollbacku, lokalnie tylko sobie na bazie cofasz migracje, czyli robisz cos typu git restore .
git checkout master
git checkout -b new-branch
I masz scheme taką jaką powinienes mieć, a jak przyjdzie Ci wracać na brancha user_roles to musisz tam po prostu migracje odpalic